### ISSUE TITLE
In the generated product.wadl, the wadl:doc tags look like this:  <_0:doc>

### ISSUE THREAD

**Issue by meker12** [rackerlabs/standard-usage-schemas#537]
Not sure if this happens in the normalized WADL, but in the generated product.wadl, the opening and closing wadl:doc tags have "0" instead of "wadl"

```
   <method id="getEntryCloudDatabaseTenant" name="GET">
           ** <_0:doc **xmlns="http://docbook.org/ns/docbook"
                    xmlns:_0="http://wadl.dev.java.net/2009/02"
                    xml:lang="EN"
                    title="Get Cloud Database Event">
```

In the feed.wadl, the tags look like this:

```
 <method id="getFeed" name="GET">
        <wadl:doc xml:lang="EN" title="Get Feed" xmlns="http://docbook.org/ns/docbook">
            <para role="shortdesc">This operation obtains the most recent events for this product's feed.</para>            
        </wadl:doc>
```
This might be the cause of the issue where the Get product event  title, parameters, and examples are missing.

**Comment by shintasmith** [comment]
@meker12, that may technically still be a valid XML. The character before the element name is just a tag/label. That tag should have the corresponding xmlns at the top. But I do see that product.wadl is missing that _0 xmlns. 

I will take a look.

**Comment by meker12** [comment]
@shintasmith  So I did some experimenting.  Changing _0 to wadl -- makes all the operations disappear completely.

One happy discovery,  the Get product-specific event by tenant ID operations are OK.  The problem was that there were two files in the RST conversion:

```get-identity-events-tid-entries-id.rst```   This file was missing the title, examples, and attributes
and 
```get-get-cloudidentity-event-identity-events-tid-entries-id.rst``` This file has all the elements, so I updated the Sphinx project to use those files.

Now we have a consistent problem with the User Access events and the Product-specific event feeds -- in that the attribute tables and examples are not formatted correctly in the RST. 


Because the allfeeds.wadl is generated from a bunch of component parts, it's pretty difficult to change the formatting to match the WADL for the other API dev guides.  But because the format is the same in these methods, I think the conversion could be scripted.  

Constanze and I can work with @j12y and with luck, the WADL2RST conversion tool can be updated to convert them cleanly.
